demo/api/schema.gql

```diff
@@ -186,6 +186,7 @@ type FileUpload {
   createdAt: DateTime!
   updatedAt: DateTime!
   downloadUrl: String
+  previewUrlTemplate: String
```

Comment:
Is this really only a "preview"? Yes, in this PR you use it as preview, but isn't that also to be used for example to show scaled User Profile Images?

And the template thing:

We have that in dam as we use those images as responsive images with a large number of different sizes, and we don't want to include all individual urls in the api response - so we create them client side using the template

Is that also the use case for FileUpload images? (alternative could be a width argument)

Reply:
> (alternative could be a width argument)

Isn't that what I'm doing? Or do you mean a Get-Parameter?

> Is this really only a "preview"? Yes, in this PR you use it as preview, but isn't that also to be used for example to show scaled User Profile Images?

You're right, there are more use cases. We could call it `imageUrl` instead.

Reply:
However, if we want to use it for scaled user profile images, they should probably be without a timeout.